starfish/pipeline/features/spots/detector/local_max_peak_finder.py

```diff
@@ -1,20 +1,24 @@
-from typing import List
+from typing import Union, Tuple
```

Comment:
Most of the code changes in this module are to unify this module with the general framework.